frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card-dropdown.tsx

```diff
@@ -47,11 +47,18 @@ export const DataConsumptionDropdown: React.FC<DataConsumptionDropdownProps> = (
     <DropdownItem
       id="egress"
       key="egress"
-      component="button"
-      isDisabled={type === typesDropdown.accounts}
-    >
+      component="button">
       Egress
+    </DropdownItem>
+  ];
+
+  const kpiDropdownItemsAccounts = [
+    <DropdownItem id="iops" key="iops" component="button">
+      I/O Operations
     </DropdownItem>,
+    <DropdownItem id="usage" key="phyVslog" component="button">
+      {type === typesDropdown.accounts ? 'Logical Usage' : 'Physical vs. Logical Usage'}
```

Comment:
I think we can use the item value directly now, since the arrays are separate. Thoughts ?

Reply:
ack